lib/document-downloader.js

```diff
@@ -112,6 +112,9 @@ DocumentDownloader.fetchAndInstall = function (url, dest) {
           extract.end(content);
         });
       }
+      else if (type && type.mime === 'application/gzip') {
+        throw new TypeError('gzip\'ed tarballs not supported!');
```

Comment:
I see these other common MIME types for compressed archives out there:
`application/x-gzip`, `application/zip`, `application/x-zip`, `application/bzip2`, `application/x-bzip2`.
Shall we check those, too?